.travis.yml

```diff
@@ -94,6 +94,11 @@ install:
   - travis_retry composer global require drupal/coder
   - $HOME/.composer/vendor/bin/phpcs --config-set installed_paths $HOME/.composer/vendor/drupal/coder/coder_sniffer
 
+  # Install Phpstan to check for Drupal standards
+  - composer global require mglaman/phpstan-drupal ^0.12.3
+  - composer global require phpstan/phpstan-deprecation-rules ^0.12.2
```

Comment:
those 2 lines can be summarized into 1. We should also run it the same way as above for webonyx/graphql-php